src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -984,12 +985,13 @@ def __call__(self, poller):
 
         try:
             result = poller.result()
-        except ClientException as client_exception:
+        except (ClientException, HttpResponseError) as client_exception:
```

Comment:
We will handle the case in anothor PR https://github.com/Azure/azure-cli/pull/17833